core/profiles/standard/standard.install

```diff
@@ -586,6 +586,7 @@ function standard_install() {
     'link_path' => 'user/login',
     'menu_name' => 'user-menu',
     'weight' => -1,
+    'customized' => 1,
```

Comment:
What's actually the purpose of this "customized" here? Can't seem to find any reference anywhere else in code.

Reply:
Menu items that are type `MENU_CALLBACK` don't appear in the menu unless they're customized. I confess I don't quite know why. But without this here, the "Login" link doesnt appear in the Account menu.